Rolique/Model/Helpers/ImageManager.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  ImageManager.swift
+//  Rolique
+//
+//  Created by Maksym Ivanyk on 9/2/19.
+//  Copyright © 2019 Rolique. All rights reserved.
+//
+
+import UIKit
+import Foundation
+
+final class ImageManager {
+  private lazy var observer: NSObjectProtocol = {
+    return NotificationCenter.default.addObserver(forName: UIApplication.didReceiveMemoryWarningNotification, object: nil, queue: nil) { [weak self] notification in
+      self?.clearImagesFolder()
```

Comment:
As I told you earlier, you don't need to clear the entire folder when app receives memory warning. Memory warning is only about dynamic memory not static. When you receive warning just clear it from `NSCache`.